node/client/src/lib.rs

```diff
@@ -72,7 +72,7 @@ where
 	Block: BlockT,
 	Backend: BackendT<Block>,
 	Backend::State: sp_api::StateBackend<BlakeTwo256>,
-	Self::Api: RuntimeApiCollection<StateBackend = Backend::State>,
```

Comment:
This change is introduced by substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)